File: app/src/main/java/com/recursia/popularmovies/utils/discover/TopRatedDiscoverStrategy.java
package com.recursia.popularmovies.utils.discover;

import com.recursia.popularmovies.utils.NetworkUtils;

public class TopRatedDiscoverStrategy implements DiscoverStrategy {

    @Override
    public String getSortBy() {
        return NetworkUtils.TOP_RATED;
    }

    @Override
    public int getVoteCount() {
        return NetworkUtils.VOTE_COUNT_TOP_RATED;
    }

}

File: app/src/main/java/com/recursia/popularmovies/utils/discover/TopRatedDiscoverStrategy.kt
package com.recursia.popularmovies.utils.discover

import com.recursia.popularmovies.utils.NetworkUtils

class TopRatedDiscoverStrategy : DiscoverStrategy {

    override val sortBy: String
        get() = NetworkUtils.TOP_RATED

    override val voteCount: Int
        get() = NetworkUtils.VOTE_COUNT_TOP_RATED
}
